web-app/prisma/schema.prisma

```diff
@@ -353,23 +354,98 @@ enum AgentListType {
   FAVORITE
 }
 
+model CreditTransaction {
+  id            String                @id @default(cuid())
+  createdAt     DateTime              @default(now())
+  updatedAt     DateTime              @updatedAt
+  amount        BigInt //positive is always a credit topup and negative is always a credit spend
+  includedFee   BigInt
```

Comment:
Does it mean that the fee is included in the `amount`?

Reply:
yes